### ISSUE TITLE
onTransitionReceived

### ISSUE THREAD

**Issue by rcastera** [cowbell/cordova-plugin-geofence#205]
Does this fire upon entering the fence? It's not clear in the documentation.

**Comment by Alexintosh** [comment]
I was wondering the same

**Comment by tsubik** [comment]
Yes, check out example applications. It may not work if your app is in the background, but this is well-known problem of Cordova applications. The most reliable way to execute custom code if app is in background is only native code https://github.com/cowbell/cordova-plugin-geofence#listening-for-geofence-transitions-in-native-code.
